qdrant-landing/content/articles/binary-quantization.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Binary Quantization - Vector Search, 40x Faster "
+short_description: "Binary Quantization is a newly introduced mechanism of reducing the memory footprint and increasing performance"
+description: "Binary Quantization is a newly introduced mechanism of reducing the memory footprint and increasing performance"
+social_preview_image: /articles_data/binary-quantization/social_preview.png
+small_preview_image: /articles_data/binary-quantization/binary-quantization-icon.svg
+preview_dir: /articles_data/binary-quantization/preview
+weight: -40
+author: Nirant Kasliwal
+author_link: 
+date: 2023-09-18T13:00:00+03:00
+draft: false
+keywords:
+  - vector search
+  - binary quantization
+  - memory optimization
+---
+
+#### Optimizing high-dimensional vectors 
+
+Qdrant is built to handle typical scaling challenges: high throughput, low latency and efficient indexing. **Binary quantization (BQ)** is our latest attempt to give our customers the edge they need to scale efficiently. This feature is particularly excellent for collections with large vector lengths and a large number of points. 
+
+Our results are dramatic: Using BQ will reduce your memory consumption and improve retrieval speeds by up to 40x.
+
+As is the case with other quantization methods, these benefits come at the cost of recall degradation. However, our implementation lets you balance the tradeoff between speed and recall accuracy at time of search, rather than time of index creation. 
+
+The rest of this article will cover:
+1. The importance of binary quantization
+2. Basic implementation using our Python client
+3. Benchmark analysis and usage recommendations
+
+![Speed vs recall of quantization and indexing methods](/articles_data/binary-quantization/bq-1.png)
```

Comment:
I see this diagram as very strange. I guess that point positions were just imagined. By idea, it looks correct. But using this diagram in practice may cause problems. Don't know what to do: add disclaimer or do another diagram option

Reply:
moved